src/render_helpers/shaders/resize.frag

```diff
@@ -5,6 +5,6 @@ vec4 resize_color(vec3 coords_curr_geo, vec3 size_curr_geo) {
     vec3 coords_tex_next = niri_geo_to_tex_next * coords_curr_geo;
     vec4 color_next = texture2D(niri_tex_next, coords_tex_next.st);
 
-    vec4 color = mix(color_prev, color_next, niri_clamped_progress);
+    vec4 color = color_mix(color_prev, color_next, niri_clamped_progress);
```

Comment:
```suggestion
    vec4 color = mix(color_prev, color_next, niri_clamped_progress);
```
This is incorrect to change, this is the window contents crossfade. There's no gradient rendering in the resize shader.

You can also remove the include from the resize shader compilation.

Actually, you can even just inline the functions into `border.frag` for now since that's the only place that draws the gradients. Thereby avoiding the whole problem.